### PR TITLE
Hw4

### PR DESCRIPTION
**Pull request by iRezograf**
Integration with Jenkins have done.  Listener didn't create. I'm working on it.

### CODE REVIEW

**Comment by iRezograf** [comment]
image

**Comment by iRezograf** [comment]
softAssert.assertAll() moved into allExerciseOne /(Two) Test.

Before/AfterClass changed on Before/AfterMethod

File: src/test/java/ru/training/at/hw4/ExerciseOne.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+    DriverManager driverManager;
+
+    @BeforeMethod(groups = {"exerciseHw41"})
+    public void setUp(ITestContext context) {
+        context.setAttribute("driver", driver);
+        softAssert = new SoftAssert();
+
+        driverManager = new DriverManager();
+        driver = driverManager.setup();
```
Comment (shoggy) [inline src/test/java/ru/training/at/hw4/ExerciseOne.java]: the driver should be placed into context after its initialization (firstly create driver, secondly put it into the context)

same for other test class

**Review by shoggy — CHANGES_REQUESTED** [review]
According to task working ex1, working ex2, and failing ex1 should exist
`@Feature` and `@Story` annotations use is required

File: src/test/java/ru/training/at/hw4/ExerciseOne.java (hunk 1 above)
Comment (iRezograf) [reply src/test/java/ru/training/at/hw4/ExerciseOne.java]: context.setAttribute("driver", driver); was moved to place after initialisation driver

@Feature and @ Story added

Comment (iRezograf) [reply src/test/java/ru/training/at/hw4/ExerciseOne.java]: image

**Review by shoggy — APPROVED** [review]

**Pull request merged**